creator-node/src/routes/nodeSync.js

```diff
@@ -24,11 +24,18 @@ module.exports = function (app) {
   app.get('/export', handleResponse(async (req, res) => {
     const start = Date.now()
 
-    // TODO - allow for offsets in the /export
     const walletPublicKeys = req.query.wallet_public_key // array
     const sourceEndpoint = req.query.source_endpoint || '' // string
 
-    const MaxClock = 25000
+    const maxExportClockValueRange = config.get('maxExportClockValueRange')
+
+    // [requestedClockRangeMin, requestedClockRangeMax] is inclusive range
+    const requestedClockRangeMin = parseInt(req.query.clock_range_min) || 0
+    const maxRequestedClockRangeMax = requestedClockRangeMin + (maxExportClockValueRange - 1)
+    const requestedClockRangeMax = Math.min((parseInt(req.query.clock_range_max) || maxRequestedClockRangeMax), maxRequestedClockRangeMax)
```

Comment:
correct, although since i wrote this i've shifted to a pattern where callers will only pass in clock_range_min so maybe this is just no longer necessary and i can rip it out?

Reply:
this is gone now